src/site-docs/apt/index.apt.vm

```diff
@@ -136,27 +136,27 @@ The <<<apache-release>>> Profile
     If you want to avoid this default assembly execution to provide your own mechanism to produce the source release archive,
     you can disable the execution by configuring <<<skipAssembly>>> parameter for this <<<source-release-assembly>>> execution id
 
- * maven-source-plugin: configured to build and attach a source jar.
+ * <maven-source-plugin>: configured to build and attach a source jar.
 
- * maven-javadoc-plugin: configured to build and attach a javadoc jar.
+ * <maven-javadoc-plugin>: configured to build and attach a javadoc jar.
 
- * maven-gpg-plugin: configured to sign everything. It expects to find
+ * <maven-gpg-plugin>: configured to sign everything. It expects to find
     a passphrase in <<<$\{gpg.passphrase\}>>>, presumably in your <<<settings.xml>>>.
 
- * checksum-maven-plugin: configured to create a checksum file(s) for source release as
+ * <checksum-maven-plugin>: configured to create a checksum file(s) for source release as
     required by {{{http://www.apache.org/dev/release-distribution#sigs-and-sums}Apache release distribution policy}}.
 
  []
 
-  When doing a release with maven-release-plugin, this creates files in <<<target/checkout/target>>> ready
+  When doing a release with <maven-release-plugin>, this creates files in <<<target>>> ready
```

Comment:
Released files are in `target/checkout/target` in `target` we have current project snapshot version.
So sentence was correct.

Reply:
You are right, this sentence refers only to releasing with m-r-p but not to the profile in general.